docs/source/ruletypes.rst

```diff
@@ -1573,6 +1573,35 @@ The alerter requires the following two options:
 ``stride_proxy``: By default ElastAlert will not use a network proxy to send notifications to Stride. Set this option using ``hostname:port`` if you need to use a proxy.
 
 
+Ryver
+~~~~~
+
+Ryver alerter will send a notification to a predefined topic in. The body of the notification is formatted the same as with other alerters. (Currently, this alerter only supports topic)
+
+The alerter requires the following options:
+
+``ryver_auth_basic``: Base64 encoded of 'user:password' for the user which will send the alert. (`Ryver does not support token yet <https://api.ryver.com/ryvrest_api_examples.html#authentication>`_ )
```

Comment:
```suggestion
``ryver_auth_basic``: Base64 encoded of ``user:password`` for the user which will send the alert. (`Ryver does not support token yet <https://api.ryver.com/ryvrest_api_examples.html#authentication>`_ )
```

Maybe we can add a small snippet to explain how to generate this?
```
python -c 'from getpass import getpass; print("{}:{}".format(getpass("Username: "), getpass("Password: ")).encode("base64"))
```

Reply:
Actually, we could probably just configure the username/password directly, it would be clearer.